src/aides.yaml

```
@@ -306,6 +306,56 @@ aides . caen:
   plafond: vélo . prix
   lien: https://caen.fr/velo-pied
 
+aides . mondeville:
+  remplace: commune
+  titre: Ville de Mondeville
+  condition de ressources: oui
+  applicable si:
+    toutes ces conditions:
+      - localisation . code postal = '14120'
+    - une de ces conditions:
+        - vélo . cargo
+        - vélo . électrique
+        - vélo
+  variations:
+    - si: vélo . électrique
+      alors:
+        grille:
+          assiette: revenu fiscal de référence
+          tranches:
+            - plafond: 620 €/mois
+              montant:
+                valeur: 300€
+            - plafond: 920 €/mois
+              montant:
+                valeur: 200€
+            - plafond: 1200 €/mois
+              montant:
+                valeur: 100€
+    - si: vélo . cargo
+      alors:
+        grille:
+          assiette: revenu fiscal de référence
+          tranches:
+            - plafond: 620 €/mois
+              montant:
+                valeur: 400€
+            - plafond: 920 €/mois
+              montant:
+                valeur: 300€
+            - plafond: 1200 €/mois
+              montant:
+                valeur: 200€
+    - si: vélo
```

`vélo` vaut toujours "oui", mieux vaut référencer `vélo . mécanique`

```suggestion
    - si: vélo . mécanique
```